exonum/tests/config.rs

```diff
@@ -32,14 +32,17 @@ use exonum::{
 use toml::Value;
 
 use std::{
+    env,
     ffi::OsString,
-    fs,
-    fs::{File, OpenOptions},
+    fs::{self, File, OpenOptions},
     io::{Read, Write},
     panic,
-    path::Path,
+    path::{Path, PathBuf},
 };
 
+const EXONUM_CONSENSUS_PASS: &str = "EXONUM_CONSENSUS_PASS";
+const EXONUM_SERVICE_PASS: &str = "EXONUM_SERVICE_PASS";
```

Comment:
Can't we somehow export these constants for tests instead of duplicating them? Or it looks too ugly?

Reply:
I looked through the Exonum code and found out that we do the same for some other constants (I don't remember which ones exactly). If I make them public, I'll have to reexport them in several modules and write documentation... Meh )

Reply:
I'm not talking about making a really 'public' public export, with docs and stuff. Something like this in the Exonum module

```rust
#[cfg(test)]
pub use module::of::the::EXONUM_SERVICE_PASS;
```

Not sure if it will work though. If it's too hard or too ugly to read then forget it.